.gitignore

```diff
@@ -12,4 +12,13 @@ target/
 bazel-bin
 bazel-ocient_jdbc
 bazel-out
-bazel-testlogs
\ No newline at end of file
+bazel-testlogs
+
+# VSCode java
+.classpath
+.project
+.settings/
+.vscode/
+
+# asdf
+.tool-versions
```

Comment:
Given that `asdf` is our recommended approach to install java, I think we ought to be committing this (in all our repos). It'll help to ensure everyone is using the same version of Java/Python/Node js.

This will become even more important as we take on projects that use different versions of the same tool. For example, lat_client and functional test already require different versions of python.

Reply:
If we do that we should start with the repos our team owns. We don't yet "own" the `ocient_jdbc` repo from an organization perspective. I'm also not sure if this belongs long-term, with the `ocient_jdbc` repo being open-source (which should typically be less prescriptive with respect to tooling).

Reply:
I agree that we ought to start with the repos our team owns.

With that said, I don't see `.tool-versions` as being prescriptive. If `asdf` isn't installed, the file simply won't do anything. If `asdf` is installed, it'll do exactly what `asdf` is supposed to, in that it'll make sure you use the right version for the project.